### PR TITLE
Show idleMinRpm on motor tab (active profile)

### PR DESCRIPTION
**Pull request by haslinghuis**
This PR adds the ability to configure either idleMinRpm (enabled and for active PID profile only) or digitalIdlePercent (disabled)  depending on Bidirectional Dshot setting. This way it makes more sense to users which setting is used.

image

image

When idleMinRpm is disabled and Bidirectional Dshot is enabled we get:

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by asizon — APPROVED** [review]
Aproved, but im not sure if this is a good approach using dynamic idle repeated in 2 diferent tabs.

**Comment by nerdCopter** [comment]
sort of makes sense to have it in motors-tab, but may lead to confusion since it is a pid-profile item. :man_shrugging:

**Comment by McGiverGim** [comment]
The idea is great, but if it's modifiable by PID profile then it has not sense to have it here.

The only solution can be to have here a "general" value, that can be overwrited for the PID profile one. If the value of the PID profile is zero, then use the general. But this needs modification in the firmware part.

If not, the best solution is to have here a "simple" text message pointing the user where he can find the idle value (in the PID profile TAB).

**Comment by nerdCopter** [comment]
adding to @McGiverGim's comment about ` "simple" text message`, potentially could also make that text show the current profile value, but not be modifiable. i.e turn the PR's implantation to a label+explanation, not a field.

**Comment by nerdCopter** [comment]
@haslinghuis , i tried to PR to your repo/branch, but PR's are disabled.
please consider potential changes:
image

```diff
diff --git a/locales/en/messages.json b/locales/en/messages.json
index b86741b9..ab1748a8 100755
--- a/locales/en/messages.json
+++ b/locales/en/messages.json
@@ -1356,7 +1356,7 @@
         "description": "Help text for the Motor poles field of the ESC/Motor configuration"
     },
     "configurationMotorIdleRpmHelp" : {
-        "message": "You will be able to edit Dynamic Idle setting on the PID tuning tab as this setting is profile dependant. When Dynamic Idle is set to zero static motor idle will apply",
+        "message": "This is the dynamic idle value from the active PID profile. When Dynamic Idle is set to zero, only static motor idle will apply. Change the Dynamic Idle settings on the PID tuning tab.",
         "description": "Help text for dynamic idle setting" 
     },
     "configurationThrottleMinimum": {
diff --git a/src/css/main.less b/src/css/main.less
index f0e1f4b0..aff27d41 100644
--- a/src/css/main.less
+++ b/src/css/main.less
@@ -103,6 +103,13 @@ input[type="number"] {
         margin-left: 5px;
     }
 }
+.noarrows {
+    input::-webkit-outer-spin-button,
+    input::-webkit-inner-spin-button {
+      -webkit-appearance: none;
+/*      margin: 0; */
+    }
+}
 .clear-both {
     clear: both;
 }
diff --git a/src/tabs/motors.html b/src/tabs/motors.html
index 8f009171..5fcf6812 100644
--- a/src/tabs/motors.html
+++ b/src/tabs/motors.html
@@ -104,7 +104,7 @@
                                 </div>
                                 <div class="number idleMinRpm">
                                     <label>
-                                        <div class="numberspacer">
+                                        <div class="numberspacer noarrows">
                                             <input type="number" name="idleMinRpm" min="0" max="100" step="1" readonly/>
                                         </div>
                                         <span i18n="pidTuningIdleMinRpm"></span>
```

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
